source-code/plugin-registry/src/schema.ts

```diff
@@ -0,0 +1,8 @@
+export type InlangPluginManifest = {
+	id: string
+	icon: string // url to an icon that can be embedded with <img src={plugin.icon}>
+	keywords: string[] // searchable keywords
+	repository: string // url, used to embed the readme
+	contributorHandle: string // github handle of the contributor
+	contributorImage: string // url to the contributor's image can be embedded with <img src={plugin.contributorImage}>
```

Comment:
Do we need contributor handle and image? Seems like repository url is enough?

Reply:
I really like to give them back reputation, like raycast is doing it. But that is just my opinion. What is the rest thinking @inlang/team

Reply:
isn't the contributor contained in the repository url? 

https://github.com/samuelstroschein/inlang-plugin-json contains samuelstroschein. https://github.com/samuelstroschein.png is my avatar

Reply:
![Uploading image.png…]()

Reply:
Jap your right as long as you are not in an organisation

Reply:
If the contributor is an organization, the organization logo should be shown no? If so, just the repo url seems enough

Reply:
Convinced -> removed it